theme_5/scss/_inputs.scss

```diff
@@ -0,0 +1,33 @@
+// Input
+.text-container {
+	border-bottom: 1px solid $cerise-red;
+	margin: 0 15px 10px;
+	.text-box {
+		// background: url(../assets/Sprite.png) no-repeat 207px -50px;
+		border: none;
+		box-shadow: none;
+		font-family: $merriweather-font;
+		font-size: 13px;
+		font-style: normal;
+		margin: 0 auto;
+		outline: none;
+		padding-left: 0;
+		text-transform: uppercase;
+	}
+}
+
+.input-container {
+	border-bottom: 1px solid $cerise-red;
+	margin-bottom: 25px;
+	.input-cu {
```

Comment:
Rule declaration should be preceded by an empty line<br>Line should be indented with spaces, not tabs